var/spack/repos/builtin/packages/phist/package.py

```diff
@@ -192,7 +192,7 @@ class Phist(CMakePackage):
     # the phist repo came with it's own FindMPI.cmake before, which may cause some other
     # MPI installation to be used than the one spack wants.
     def patch(self):
-        if self.spec.satisfies("@1.9.6"):
+        if self.spec.satisfies("@1.9.6:"):
```

Comment:
This patch undoes the switch from the unsafe 'mpi' to the better 'mpi_f08' module which is unfortunate. The actual fix should be to make it possible to query wether an mpi spec provides mpi_f08, and patch phist only if it doesn't.
Still I think this is OK for now.

Reply:
This PR is intended as a stopgap to allow the CI to continue functioning without error, but thank you for pointing this out. I'll leave a TODO to indicate that this should not languish in its current state.